eventfiles/class_tests/event_file_views_tests.py

```diff
@@ -46,4 +61,47 @@ def test_unauthorized_user_redirected_to_main_page(self, client):
         assert response.status_code == 302
         response = client.get(response.url)
         assert response.status_code == 200
-        assertTemplateUsed(response, 'main/index.html')
+        assertTemplateUsed(response, 'login/register_login.html')
```

Comment:
I think you should change the function name to `test_unauthorized_user_redirected_to_login_page` instead of `test_unauthorized_user_redirected_to_main_page`

Reply:
Done